share/completions/bzr.fish

```diff
@@ -1,604 +1,156 @@
+# TODO: Add source/destination branch or directory
+
+# add command
+complete -f -c bzr -n '__fish_bzr_needs_command' -a add -d 'Make files or directories versioned.'
+complete -f -c bzr -n '__fish_bzr_using_command add' -l no-recurse -s N -d 'Don\'t recursively add the contents of directories'
+
+# ignore command
+complete -f -c bzr -n '__fish_bzr_needs_command' -a ignore -d 'Ignore a file or pattern.'
+complete -f -c bzr -n '__fish_bzr_using_command ignore' -l default-rules -d 'Display the default ignore rules that bzr uses.'
+
+# mv command
+complete -f -c bzr -n '__fish_bzr_needs_command' -a mv -d 'Move or rename a versioned file.'
+complete -f -c bzr -n '__fish_bzr_using_command mv' -l auto -d 'Automatically guess renames.'
+complete -f -c bzr -n '__fish_bzr_using_command mv' -l after -d 'Move only the bzr identifier of the file, because the file has already been moved.'
+
+# status command
+complete -f -c bzr -n '__fish_bzr_needs_command' -a status -d 'Summarize changes in working copy.'
+complete -f -c bzr -n '__fish_bzr_using_command status' -l short -s S -d 'Use short status indicators.'
+complete -f -c bzr -n '__fish_bzr_using_command status' -l versioned -s V -d 'Only show versioned files.'
+complete -f -c bzr -n '__fish_bzr_using_command status' -l no-pending -d 'Don\'t show pending merges.'
+complete -f -c bzr -n '__fish_bzr_using_command status' -l no-classify -d 'Do not mark object type using indicator.'
+complete -f -c bzr -n '__fish_bzr_using_command status' -l show-ids -d 'Show internal object ids.'
+
+# diff command
+complete -f -c bzr -n '__fish_bzr_needs_command' -a diff -d 'Show detailed diffs.'
+# TODO: Add file diff options
+
+# merge command
+complete -f -c bzr -n '__fish_bzr_needs_command' -a merge -d 'Pull in changes from another branch.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l pull -d 'If the destination is already completely merged into the source, pull from the source rather than merging.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l remember -d 'Remember the specified location as a default.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l force -d 'Merge even if the destination tree has uncommitted changes.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l reprocess -d 'Reprocess to reduce spurious conflicts.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l uncommited -d 'Apply uncommitted changes from a working copy, instead of branch changes.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l show-base -d 'Show base revision text in conflicts.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l preview -d 'Instead of merging, show a diff of the merge.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l interactive -s i -d 'Select changes interactively.'
+complete -f -c bzr -n '__fish_bzr_using_command merge' -l directory= -s d -d 'Branch to merge into, rather than the one containing the working directory.'
```

Comment:
Remove the "=". We always allow "--option=argument" and "--option argument" for long options.